.prettierrc.yaml

```diff
@@ -0,0 +1,10 @@
+printWidth: 80
```

Comment:
Not changing this as it requires reformatting the files, just want to add a config for now.